docker/build/mule.go.centos.Dockerfile

```diff
@@ -0,0 +1,32 @@
+ARG ARCH="amd64"
```

Comment:
Why couldn't you use the centos dockerfile we're using for the build process?

Reply:
I need mule.  See the comment above.  Please suggest an alternative.